src/pages/serverStatus/ServerStatus.jsx

```diff
@@ -11,14 +11,15 @@ import { useTheme } from '@material-ui/core/styles';
 import version from '../../constants/version';
 import useGetSiteInfo from '../../models/site/useGetSiteInfo';
 import useDocumentTitle from '../../hooks/useDocumentTitle';
-import useServerStatus from '../../models/server/useServerStatus';
```

Comment:
is useServerStatus now unused? can we delete it?

Reply:
It ended up getting moved to `src/utils/sageUtils.js`.
```
src/models/server/useServerStatus.js → src/utils/sageUtils.js
```

Reply:
oh nice